verification/curator-service/ui/src/components/App.tsx

```diff
@@ -304,8 +304,7 @@ export default function App(): JSX.Element {
             text: 'Linelist',
             icon: <ListIcon />,
             to: { pathname: '/cases', state: { search: '' } },
-            displayCheck: (): boolean =>
-                hasAnyRole(['reader', 'curator', 'admin']),
+            displayCheck: (): boolean => user.email !== '',
```

Comment:
Is it `''` or `undefined` when a user isn't logged in? Could this just be `displayCheck: (): boolean => user.email` ?

Reply:
the default value for user in the app component is an empty user with all fields set to their default value so I have to check for email here (it's done as well like that below for other parts of the App component), we could change to undefined but I'd prefer to do that in another PR.